pkgs/development/compilers/rml/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, ocamlPackages
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rml";
+  version = "1.09.07";
+
+  src = fetchFromGitHub {
+    owner = "reactiveml";
+    repo = pname;
+    rev = "${pname}-${version}-2021-07-26";
+    hash = "sha256-UFqXQBeIQMSV4O439j9s06p1hh7xA98Tu79FsjK9PIY=";
+  };
+
+  buildInputs = with ocamlPackages; [
+    ocaml
```

Comment:
Isn't ocaml run from PATH? Then it goes to nativeBuildInputs

Reply:
Done.